### PR TITLE
fix bug where user is displayed

### PR DESCRIPTION
**Pull request by nvergel**

### CODE REVIEW

File: code/src/main/java/com/onlinecontacttracing/servlets/get-data/PositiveUserInfoServlet.java [hunk 1]
```diff
@@ -54,11 +53,13 @@ void useCredential(AuthorizationRoundTripState state, Credential credential, Htt
     calendarInfo.join();
 
     PositiveUserContacts fullContacts = ofy().load().type(PositiveUserContacts.class).id(state.userId).now();
-    if(fullContacts.getListOfContacts().isEmpty()) {
+    if(fullContacts == null) {
```
Comment (bengaskagoogle) [inline code/src/main/java/com/onlinecontacttracing/servlets/get-data/PositiveUserInfoServlet.java:56]: Do you want to check for both the null case and the empty contact list case?

**Review by bengaskagoogle — APPROVED** [review]

File: code/src/main/java/com/onlinecontacttracing/servlets/get-data/PositiveUserInfoServlet.java (hunk 1 above)
Comment (nvergel) [reply code/src/main/java/com/onlinecontacttracing/servlets/get-data/PositiveUserInfoServlet.java:56]: It doesn't matter if it is empty, the set will still add data.

Comment (nvergel) [reply code/src/main/java/com/onlinecontacttracing/servlets/get-data/PositiveUserInfoServlet.java:56]: Also in the runnable that populates that list, it checks if the list is empty before saving so they should be mutually exclusive.

Comment (bengaskagoogle) [reply code/src/main/java/com/onlinecontacttracing/servlets/get-data/PositiveUserInfoServlet.java:56]: Sounds good to me. Feel free to resolve.

Comment (nvergel) [reply code/src/main/java/com/onlinecontacttracing/servlets/get-data/PositiveUserInfoServlet.java:56]: meant to say mutually inclusive*

**Pull request merged**